client/signup/jetpack-connect/plans.jsx

```diff
@@ -58,12 +58,11 @@ const Plans = React.createClass( {
 	},
 
 	componentWillReceiveProps( props ) {
-		const selectedSite = props.sites.getSelectedSite();
```

Comment:
Is there any particular reason to use the old props in `componentWillReceiveProps` instead of the new ones (as it was before)?